docs/reference/cat/nodes.asciidoc

```diff
@@ -43,9 +43,11 @@ Valid columns are:
 
 `node.role`, `r`, `role`, `nodeRole`::
 (Default) Roles of the node. Returned values include `m` (master-eligible node),
-`d` (data node), `i` (ingest node), `v` (voting-only node), and `-` (coordinating node only).
+`d` (data node), `i` (ingest node), `l` (machine learning node), `v`
+(voting-only node), and `-` (coordinating node only).
 +
-For example, `mdi` indicates a master-eligible data and ingest node.
+For example, `mdi` indicates a master-eligible data and ingest node. See
```

Comment:
I think the letters are now sorted alphabetically, so `mdi` would be shown as `dim`.

Reply:
You're right.  I was hoping it would say "mild", but no such luck :)